### PR TITLE
#44 get endpoint for project primary skills

### PR DESCRIPTION
**Pull request by alesyalis**

### CODE REVIEW

File: Recrutify/Recrutify.Host/Controllers/ProjectsController.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ public ProjectsController(IProjectService projectService, IPrimarySkillService p
 
         [Authorize(Policy = Constants.Policies.AdminPolicy)]
         [HttpGet("primary_skills")]
-        public async Task<ActionResult<List<PrimarySkillDTO>>> GetPrimarySkillAsync()
+        public async Task<ActionResult<List<PrimarySkillDTO>>> GeAlltPrimarySkillAsync()
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Host/Controllers/ProjectsController.cs:30]: GeAllPrimarySkillsAsync

Comment (alesyalis) [reply Recrutify/Recrutify.Host/Controllers/ProjectsController.cs:30]: done

File: Recrutify/Recrutify.Services/Services/PrimarySkillService.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿using System.Collections.Generic;
+﻿using System;
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Services/Services/PrimarySkillService.cs:1]: remove extra usings

Comment (alesyalis) [reply Recrutify/Recrutify.Services/Services/PrimarySkillService.cs:1]: done

**Review by elizavetapavliv — APPROVED** [review]

**Pull request merged**